services/manager/lib/io/http/index.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const ip = require('ip');
+const pathLib = require('path');
+
+const mountApp = ({ server, name, path }) => {
+  const mountAt = `/${name}`;
+
+  server.use(mountAt, express.static(path));
+};
+
+const mountAppList = (apps, server) => {
+  const appNames = apps.map( a => a.name );
```

Comment:
This used to be `{ id: a.name }`. It's simpler to return the array but it means [this line](https://github.com/andrewn/neue-radio/blob/80282935415ac77b27cefe52d07e689e871151fe/services/manager/public/index.html#L30-L32) needs to change to:

```js
url.pathname = app;
const id = app;
```